src/Microsoft.Extensions.Primitives/project.json

```diff
@@ -1,11 +1,14 @@
 {
   "version": "1.0.0-*",
-  "description": "ASP.NET 5 primitives.",
+  "description": "ASP.NET Core primitives. Commony used types include:\r\nMicrosoft.Extensions.Primitives.IChangeToken\r\nMicrosoft.Extensions.Primitives.StringValues\r\nMicrosoft.Extensions.Primitives.StringSegment",
   "packOptions": {
     "repository": {
       "type": "git",
       "url": "https://github.com/aspnet/common"
-    }
+    },
+    "tags": [
+      "primitives"
```

Comment:
@Eilon's work item said

> Note: Certain packages are not part of ASP.NET Core or Entity Framework Core, so be careful not to add those tags to the metadata of these packages.

This seemed like a candidate seeing we don't tag configuration or fileproviders with `aspnetcore`